### PR TITLE
14184 display examiner

### PR DESCRIPTION
**Pull request by ozamani9gh**
*Issue #:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/static/keycloak/keycloak.json
Comment (bolyachevets) [inline client/static/keycloak/keycloak.json:1]: should we use prod keycloak url here: https://loginproxy.gov.bc.ca/auth/

Comment (ozamani9gh) [reply client/static/keycloak/keycloak.json:1]: i dont know

Comment (ozamani9gh) [reply client/static/keycloak/keycloak.json:1]: After chatting with Richard, this file does not need to change, its not used in prod.    please approve

**Comment by rarmitag** [comment]
I believe Keycloak.json is always loaded. The others are there to rename when you were pointing at dev or test.

**Pull request merged**